app/setupTests.js

```diff
@@ -7,3 +7,27 @@ import Adapter from 'enzyme-adapter-react-16';
 
 // Create Enzyme adapter.
 configure({ adapter: new Adapter() });
+
+// Mock localStorage
+// eslint-disable-next-line flowtype/no-weak-types
+const localStorageMock = (): Object => {
+// eslint-disable-next-line flowtype/no-weak-types
+  let store: Object = {};
```

Comment:
Please don't use the `Object` type or other weak types if it can be avoided (see [here](https://flow.org/en/docs/types/objects/#toc-object-type) and [here](https://flow.org/en/docs/types/any/) for reasons). In this case, Flow can infer the return type of the `localStorageMock` function directly from its code, so you can use `*` as its return type. The store object can be typed as `{ [key: string]: string }` to indicate it's a map of string keys to string values.